reference/configuration/swiftmailer.rst

```diff
@@ -191,6 +225,13 @@ logging
 If true, Symfony's data collector will be activated for Swift Mailer and
 the information will be available in the profiler.
 
+.. tip::
+
+    The following configuration can used environment variable:
```

Comment:
The following configuration options are compatible with `%env()%` parameters (with link to doc about them?)